extensions/amp-story/1.0/amp-story-reaction-quiz.js

```diff
@@ -95,7 +96,7 @@ export class AmpStoryReactionQuiz extends AmpStoryReaction {
    * @param {Element} root
    */
   attachContent_(root) {
-    // TODO(jackbsteinberg): Optional prompt behavior must be implemented here
+    // Configure header.
```

Comment:
Ok, I wasn't sure about that since it seems like there are tests that are supposed to fail if no header is present, and the previous quiz implementation didn't allow for no header to be added. Will add that.